### PR TITLE
More filters: Bug fixes changes and text changes 

### PR DESCRIPTION
**Pull request by ishouty**
* move contributors ordering in mobile and desktop
* update mobile/desktop checkbox ui styling when text is long
* renaming of title text
* overriding styles for checkbox ui

### CODE REVIEW

File: common/views/components/CheckboxRadio/CheckboxRadio.tsx [hunk 1]
```diff
@@ -70,24 +70,36 @@ type CheckboxRadioProps = {
...
 };
 
+export type BaseLabelContainerAttr = {
+  as?: string;
+};
+
+export const BaseLabelContainer = styled(Space).attrs<BaseLabelContainerAttr>(
+  () => ({
+    h: { size: 'xs', properties: ['margin-left'] },
+    as: 'label',
```
Comment (jamesgorrie) [inline common/views/components/CheckboxRadio/CheckboxRadio.tsx:83]: Why do we need these `attrs` if it's hardcoded?

File: common/views/components/ModalMoreFilters/ModalMoreFilters.tsx [hunk 2]
```diff
@@ -112,6 +114,16 @@ const FiltersHeader = styled(Space).attrs({
   width: 100%;
 `;
 
+const OverrideLabelContainer = styled(BaseLabelContainer)`
```
Comment (jamesgorrie) [inline common/views/components/ModalMoreFilters/ModalMoreFilters.tsx:117]: Why do we need this override? Could we leave a note so we remember the reason when we come back to this in the future.

Comment (ishouty) [reply common/views/components/ModalMoreFilters/ModalMoreFilters.tsx:117]: Agreed, not the right solution for this and as discussed implement a universal solution.

File: common/views/components/CheckboxRadio/CheckboxRadio.tsx [hunk 3]
```diff
@@ -61,6 +61,11 @@ const CheckboxRadioInput = styled.input.attrs(props => ({
   }
 `;
 
+const CheckBoxWrapper = styled.div`
+  position: relative;
+  top: 1px;
```
Comment (jamesgorrie) [inline common/views/components/CheckboxRadio/CheckboxRadio.tsx:66]: Could you leave a note explaining why this is here?

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**